packages/hardhat-verify/README.md

```diff
@@ -4,6 +4,20 @@
 
 [Hardhat](https://hardhat.org) plugin to verify the source of code of deployed contracts.
 
+1. [What](#what)
+2. [Installation](#installation)
+3. [Tasks](#tasks)
+4. [Environment extensions](#environment-extensions)
+5. [Usage](#usage)
+6. [Complex arguments](#complex-arguments)
+7. [Libraries with undetectable addresses](#libraries-with-undetectable-addresses)
+8. [Multiple API keys and alternative block explorers](#multiple-api-keys-and-alternative-block-explorers)
+9. [Adding support for other networks](#adding-support-for-other-networks)
+10. [Using programmatically](#using-programmatically)
+11. [Advanced Usage: Using the Etherscan class from another plugin](#advanced-usage-using-the-etherscan-class-from-another-plugin)
+12. [How it works](#how-it-works)
+13. [Known limitations](#known-limitations)
```

Comment:
Can we use nesting in the proper sub-sections? For example, "Complex arguments" is a sub-section of "Usage", and 11 and 12 are sub-sections of "Using programmatically", which in turn is also a sub-section of "Usage".